docs/modules/ROOT/pages/how-to-guides/managing-environments/proc_creating_your_own_environment.adoc

```diff
@@ -1,31 +1,63 @@
 = Creating your own environment 
 
-Use {ProductName} to continuously build and test your application. Use your own cluster as your deployment environment. By using your own cluster outside Red Hat, you have more control over where your application is deployed, and you can ensure it meets your specific needs. If you bring your own cluster you can still access all the features and benefits of {ProductName}. 
+You can configure {ProductName} to deploy applications to your own cloud-hosted cluster, outside the default development environment. By using this option to bring your own cluster (BYOC), you can deploy your applications to an environment that meets your specific needs. And if you bring your own cluster, you can still access all the features and benefits of {ProductName}. 
 
 .Prerequisites
 
-* You must have access to credentials with full read and write access to the namespace that you provided to {ProductName}.   
-* When you upload a `kubeconfig` file, it must include a credential with full read and write access to your provided namespace. 
-* The supplied `kubeconfig` file must use token-based authentication to connect to your cluster. Other mechanisms, such as certificate-based authentication, are not yet supported. 
-* The context in your `kubeconfig` file must point to the cluster you want to use at the time you upload the file. 
-
-.Procedures
-
-To create your own environment and upload your own `kubeconfig` file, complete the following steps in the {ProductName} UI: 
-
-. Go to *Environments* > *Create Environment*. 
-. Complete all the required fields for *Create Environment*.  
+* You must have credentials for full read and write access to the namespace for your cluster.   
+
+.Procedure
+
+To create your own environment, you first need to create a functional `kubeconfig` file. Then, using that file, you add the cluster as a new environment in the {ProductName} UI.
+
+. In your preferred text editor, create a `kubeconfig` file based on the following example:
```

Comment:
Suggestion: Since we have an example below for additional clarity can we add annotations in the example? For example, `server` - What kind of server, from where to get the link?
same for `cluster`, and `namespace` (What kind of namespace?)
... and others

For example, in the link look for annotations in Procedure step 2. https://pr-197--boisterous-meerkat-894982.netlify.app/documentation/main/how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build/

Reply:
I think that's a good suggestion, so I added quite a few annotations, and tried to be specific. That having been said, there are some details that users might have to figure out on their own, particularly depending on the cloud service they're using to host their cluster. Anyway, please check out the updated preview and see what you think about the annotations:
https://pr-199--boisterous-meerkat-894982.netlify.app/documentation/main/how-to-guides/managing-environments/proc_creating_your_own_environment/